packages/okta-react/test/e2e/harness/package.json

```diff
@@ -29,10 +29,10 @@
     "start": "REACT_APP_ISSUER=$ISSUER REACT_APP_CLIENT_ID=$SPA_CLIENT_ID PORT=8080 OKTA_TESTING_DISABLEHTTPSCHECK=$OKTA_TESTING_DISABLEHTTPSCHECK react-app-rewired start",
     "build": "REACT_APP_ISSUER=$ISSUER REACT_APP_CLIENT_ID=$SPA_CLIENT_ID PORT=8080 OKTA_TESTING_DISABLEHTTPSCHECK=$OKTA_TESTING_DISABLEHTTPSCHECK react-app-rewired build",
     "build:test": "rimraf e2e/dist && babel e2e/ -d e2e/dist",
-    "kill:port": "kill -9 $(lsof -t -i:8080 -sTCP:LISTEN) || true",
-    "pretest": "yarn kill:port && ../../../../../scripts/updateSeDrivers.sh 0 && yarn build:test",
+    "kill:port": "kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN) || true",
```

Comment:
I was fighting this same thing, finding a port check that worked locally as well as on travis.  *yoink*

Reply:
It caused me a lot of confusion locally. The zombie web server was left running in the background. I had no idea.  Using 'TERM' (instead of SIGKILL) for the signal allows it to exit gracefully with exit code 0